nexus/src/authz/omicron.polar

```diff
@@ -178,3 +178,38 @@ resource Project {
 }
 has_relation(organization: Organization, "parent_organization", project: Project)
 	if project.organization = organization;
+
+resource GlobalImageList {
+	permissions = [
+	    "list_children",
+	    "modify",
+	    "create_child",
+	];
+
+	# Only admins can create or modify the global images list
+	relations = { parent_fleet: Fleet };
+	"modify" if "admin" on "parent_fleet";
+	"create_child" if "admin" on "parent_fleet";
+
+	# Anyone with viewer can list global images
+	"list_children" if "viewer" on "parent_fleet";
+}
+has_relation(fleet: Fleet, "parent_fleet", global_image_list: GlobalImageList)
+	if global_image_list.fleet = fleet;
+
+resource GlobalImage {
```

Comment:
I would consider _not_ writing this into omicron.polar but just using the `FleetChild` generation option to `authz_resource!`.  For each of the permissions, rules, and relations you've defined here, that option defines them with the same behavior.  It defines some additional things that you don't need here, but I think a single source of implementation is worth it.

Reply:
:+1: done in a52de65b